tensorflow_datasets/core/lazy_imports.py

```diff
@@ -102,6 +102,11 @@ def skimage(cls):
     _try_import("skimage.external.tifffile")
     return _try_import("skimage")
 
+  @utils.classproperty
+  @classmethod
+  def tfdv(cls):
+    return _try_import("tensorflow_data_validation")
```

Comment:
You would need to edit setup.py, no?

Reply:
Etienne suggest the add tfdv as a lazy import due to its size.
`• As tfdv is a big dependency, and create some issues on some system (ex: Colab). Make sure that tfdv is used as lazy dependency. As statistics are not essential to the program, you can skip the statistics computation if tfdv isn't installed.
`

Reply:
Even if it is added as lazy deps, external users shouldn't encounter those issues which are Colab and Google internal specific.
So it should be safe to add it in setup.py